detroit-labs/redux-devtools-inspector
# Fixing order of type checking and value reassignment

### Description

This is a PR to fix [an issue](https://github.com/detroit-labs/lineage-frontend/issues/992) we are seeing on the MetricsOne frontend.

This issue is currently up in a [PR](https://github.com/alexkuz/redux-devtools-inspector/pull/72) to be fixed on the `redux-devtools-inspector`  repo.  However, it has been up since May, and there is no resolution in sight.

[There is also an issue up for the original repo.](https://github.com/alexkuz/redux-devtools-inspector/issues/73)

### Testing Steps

In order to test this, you will need to pull this branch on `redux-devtools-inspector` into your FE.

1.  While on the FE, run `yarn remove redux-devtools-inspector`.
2.  After the command successfully finishes, run ` yarn add github:detroit-labs/redux-devtools-inspector#LNG-2305-Fixing-typechecking-order-issue`
3.  Once this command successfully finished, start up the FE and the MW locally.
4.  Ensure that the FE compiles correctly.
5.  Click around through the FE to make sure the app works.
6.  If you were not having the issue happen, this is the full test.  Please see me for more info if you have questions..
7.  If you were having the issue, this should resolve it 🤞 .


Once this PR is approved and merged in, I will put up a corresponding FE PR.
